graph-db/src/utils/bootstrapGremlinServer.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-await-in-loop */
```

Comment:
It's best to use these disables on the line that you are disabling for tracking. If you end up 'fixing' this and its no longer necessary you want to remove this, and having it right next to the line of code being changed helps ensure that happens.

Reply:
+1

Reply:
Good call - fixed